src/nfts/NftMedia.tsx

```diff
@@ -12,23 +14,30 @@ import variables from 'src/styles/variables'
 import Logger from 'src/utils/Logger'
 import networkConfig from 'src/web3/networkConfig'
 
-const DEFAULT_IMAGE_HEIGHT = 360
+const DEFAULT_HEIGHT = 360
 
-interface ImagePlaceHolderProps {
+interface PlaceHolderProps {
   testID: string
   height?: number
   width?: number
   borderRadius?: number
+  mediaType: 'image' | 'video'
 }
 
-interface Props extends ImagePlaceHolderProps {
+interface Props extends PlaceHolderProps {
   nft: Nft
   origin: NftOrigin
   shouldAutoScaleHeight?: boolean
   ErrorComponent: React.ReactNode
 }
 
-function ImagePlaceholder({ height = 40, width, borderRadius = 0, testID }: ImagePlaceHolderProps) {
+function Placeholder({
+  testID,
+  width,
+  mediaType,
+  height = mediaType === 'video' ? DEFAULT_HEIGHT : 40,
```

Comment:
Why such a big difference between video and images for the height here?

Reply:
The video only is present in the main image section of the carousel, but the images are used as thumbnails several places and use a smaller default height of 40 in those places.

Reply:
I see, maybe it's fine, but my reaction was it could be surprising as a developer to get a different height when not specified and just the media type changes.

Reply:
I think we should use the default height here. And force all the clients of this component to specify the height of 40 if they need to.

Reply:
Makes sense. I checked an in most places we specify 40 where needed so I'll set it to `DEFAULT_HEIGHT` here.